core/src/main/java/pl/agh/edu/GdxGame.java

```diff
@@ -38,7 +39,9 @@ public void render() {
 			currentScreen.render(Gdx.graphics.getDeltaTime());
 		}
 
-		engine.time.update(Gdx.graphics.getDeltaTime());
+		if (engine != null) {
```

Comment:
o twrzenie engine chyba bym zostawił na innego PR , bo tak będziemy musieli to jeszcze połączyć z serializacją a w ten sposób i tak nie bedziemy robić